### PR TITLE
Adding Droids4Droids

### PR DESCRIPTION
**Pull request by ykro**
Entries for Droids 4 Droids Lima & Guatemala

### CODE REVIEW

File: _conferences/droids4droids-gt-2017.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: "Droids 4 Droids"
+website: http://gt.droids4droids.com/
+location: Guatemala, GT
+
+date_start: 2017-10-18
+date_end:   2017-10-18
+
+cfp_site: https://goo.gl/forms/4uWoC1e29P5Jorbp2
```
Comment (JakeWharton) [inline _conferences/droids4droids-gt-2017.md]: This won't show up without a `cfp_start` and `cfp_end` date.

File: _conferences/droids4droids-gt-2017.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: "Droids 4 Droids"
+website: http://gt.droids4droids.com/
+location: Guatemala, GT
```
Comment (JakeWharton) [inline _conferences/droids4droids-gt-2017.md]: I don't quite understand this location.. shouldn't it be "Something, Guatemala"?

Comment (ykro) [reply _conferences/droids4droids-gt-2017.md]: fixed, it's the same name for city and country.

File: _conferences/droids4droids-gt-2017.md (hunk 1 above)
Comment (ykro) [reply _conferences/droids4droids-gt-2017.md]: fixed

**Comment by JakeWharton** [comment]
Thanks!

**Pull request merged**